### ISSUE TITLE
notify when the CLI is out-of-date

### ISSUE THREAD

**Issue by kenneth-reitz** [storyscript/cli#171]

**Comment by kenneth-reitz** [comment]
npm did this today:

```

   ╭───────────────────────────────────────────────────────────────╮
   │                                                               │
   │       New minor version of npm available! 6.4.1 → 6.9.0       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.9.0   │
   │               Run npm install -g npm to update!               │
   │                                                               │
   ╰───────────────────────────────────────────────────────────────╯
```

I might try to emulate that.

**Comment by kenneth-reitz** [comment]
Nah, I'll just do a simple line. This is very invasive.
